src/main/java/org/jitsi/jigasi/lobby/Lobby.java

```diff
@@ -0,0 +1,213 @@
+package org.jitsi.jigasi.lobby;
```

Comment:
Missing license header.

Reply:
And in general missing javadocs for this class.

Reply:
Added javadocs.